examples/strings/strings.bal

```diff
@@ -22,14 +22,34 @@ public function main() {
     if (index is int) {
         io:println("IndexOf: ", index);
     }
-    
+
     // Retrieves the length of the `string`.
     int length = statement.length();
     io:println("Length: ", length);
 
+    string hello = "Hello";
+    string ballerina = "Ballerina!";
+
+    // Concats multiple strings.
+    string s4 = strings:concat(hello, " ", ballerina);
```

Comment:
Shall we change this as follows as discussed offline?

```suggestion
    string s4 = hello.concat(" ", ballerina);
```

Let's import langlibs only if it is not possible to do - as above - without the import. Please check other changes too.